weather_dl/download_pipeline/manifest.py

```diff
@@ -106,76 +202,161 @@ class Manifest(abc.ABC):
 
     def __post_init__(self):
         """Initialize the manifest."""
-        self.start = 0
-        self.scheduled_times = {}
+        pass
```

Comment:
umm this is required because we are using __post_init__() in ConsoleManifest (inherited from our base Manifest dataclass, defining __post_init__() in just the child class seems to be not working w.r.t to dataclasses). I do have a cleaner solution for the same.

**BUT** as per this approach of manifest, we can't really use ConsoleManifest because we need to read the state from the DB before updating the manifest.
Hence I was thinking of getting rid of ConsoleManifest all together and making LocalManifest as the default manifest system when running the tool in the local system & BQ manifest as the default manifest system when running the tool in Dataflow runner.
Furthermore we anyways log the manifest on the console after writing it so ConsoleManifest will be there always.

WDYT @alxmrs ?

Reply:
I think we should keep the console manifest. Even when we're running on Dataflow, we want the tool to not have a huge amount of infrastructural dependencies to get up and running. i.e. to get started, a new user would have to set up one additional GCP service (BigQuery) to use the tool. @blackvvine can explain this reasoning better than I can: 
https://github.com/google/weather-tools/issues/246#issue-1401678061

What you're saying makes sense wrt to keeping post_init. I'm happy to keep it -- can you just add a comment explaining that this is used in the subclasses?

Reply:
> BUT as per this approach of manifest, we can't really use ConsoleManifest because **we need to read the state from the DB before updating the manifest.**

Reply:
As discussed in a 1:1 today, the console manifest doesn't have to be stateful (or perfect). We can merely update the user with the information we have available.